src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml

```diff
@@ -24,54 +24,54 @@ limitations under the License.
+    <attribute side="server" code="0x0001" define="CHANNEL_LINEUP"          type="LineupInfoStruct"                    default="0x0" isNullable="true" writable="false" optional="true">Lineup</attribute>
+    <attribute side="server" code="0x0002" define="CHANNEL_CURRENT_CHANNEL" type="ChannelInfoStruct"                   default="0x0" isNullable="true" writable="false" optional="true">CurrentChannel</attribute>
 
     <command source="client" code="0x00" name="ChangeChannel" response="ChangeChannelResponse" optional="true">
       <description>Change the channel on the media player to the channel case-insensitive exact matching the value passed as an argument. </description>
-      <arg name="match" type="CHAR_STRING"/>
+      <arg name="Match" type="CHAR_STRING"/>
     </command>
 
     <command source="client" code="0x02" name="ChangeChannelByNumber" optional="false">
       <description>Change the channel on the media plaeyer to the channel with the given Number in the ChannelList attribute.</description>
-      <arg name="majorNumber" type="INT16U"/>
-      <arg name="minorNumber" type="INT16U"/>
+      <arg name="MajorNumber" type="INT16U"/>
+      <arg name="MinorNumber" type="INT16U"/>
     </command>
 
     <command source="client" code="0x03" name="SkipChannel" optional="false">
       <description>This command provides channel up and channel down functionality, but allows channel index jumps of size Count. When the value of the increase or decrease is larger than the number of channels remaining in the given direction, then the behavior SHALL be to return to the beginning (or end) of the channel list and continue. For example, if the current channel is at index 0 and count value of -1 is given, then the current channel should change to the last channel.</description>
-      <arg name="count" type="INT16U"/>
+      <arg name="Count" type="INT16U"/>
     </command>
 
     <command source="server" code="0x01" name="ChangeChannelResponse" optional="true">
       <description>Upon receipt, this SHALL display the active status of the input list on screen.</description>
-      <arg name="status" type="ChannelStatusEnum"/>
-      <arg name="data"   type="CHAR_STRING" optional="true"/>
+      <arg name="Status" type="ChannelStatusEnum"/>
+      <arg name="Data"   type="CHAR_STRING" optional="true"/>
     </command>
 
   </cluster>
 
-  <struct name="ChannelInfo">
+  <struct name="ChannelInfoStruct">
     <cluster code="0x0504"/>
-    <item name="majorNumber"       type="INT16U"/>
-    <item name="minorNumber"       type="INT16U"/>
-    <item name="name"              type="CHAR_STRING" optional="true" length="32"/>
-    <item name="callSign"          type="CHAR_STRING" optional="true" length="32"/>
-    <item name="affiliateCallSign" type="CHAR_STRING" optional="true" length="32"/>
+    <item name="MajorNumber"       type="INT16U"/>
```

Comment:
same here: minor and major in the spec are lowercase, however all the rest are uppercase.

Reply:
Spec has been updated recently.